030_elm_arch/actions.md

```diff
@@ -196,7 +196,7 @@ main =
 
 <https://github.com/sporto/elm-tutorial-assets/blob/master/code/elm_arch/ActionsMultiple.elm>
 
-Every mouse click increases the count, every key press decreases the count. Note how we __merge__ the signals (mouseClickSignal and keyPressSignal) into one, as they both are of type `Signal.Signal Action` our folp can handle them.
+Every mouse click increases the count, every key press decreases the count. Note how we __merge__ the signals (mouseClickSignal and keyPressSignal) into one, as they both are of type `Signal.Signal Action` our foldp can handle them.
```

Comment:
I was actually tempted to change it to:

```
"...as they both are of type `Signal.Signal Action`, `Signal.foldp` can handle them."
```

But then I went with the minimal fix.

@sporto thoughts?